revolv/base/forms.py

```diff
@@ -21,6 +21,7 @@ def save(self, commit=True):
         last names.
         """
         user = super(SignupForm, self).save(commit=False)
+        user.save()
```

Comment:
Hm, what's the error? If the user needs to be saved before setting its attributes, then how did it ever work before?